providers/implementations/signature/ecdsa_sig.c

```diff
@@ -377,81 +531,99 @@ static int ecdsa_digest_sign_init(void *vctx, const char *mdname, void *ec,
                                   const OSSL_PARAM params[])
 {
     return ecdsa_digest_signverify_init(vctx, mdname, ec, params,
-                                        EVP_PKEY_OP_SIGN,
+                                        EVP_PKEY_OP_SIGNMSG,
                                         "ECDSA Digest Sign Init");
 }
 
-static int ecdsa_digest_verify_init(void *vctx, const char *mdname, void *ec,
-                                    const OSSL_PARAM params[])
+static int ecdsa_digest_sign_update(void *vctx, const unsigned char *data,
```

Comment:
Looks like a copy & paste of ecdsa_digest_verify_update

Reply:
Already noted, already reversed